basics/Strings examples/replace vowels.py

```diff
@@ -0,0 +1,8 @@
+str=input("Type the string:")
+str1=""
+for i in str:
+    if i in "aeiouAEIOU":
+        i="*"
+    str1=str1+i

```

Comment:
This could be turned into a good "simplify this code" exercise. It can be simplified a lot by instead using `.replace()`:

```python3
for wovel in "aeiouAEIOU":
    string = string.replace(wovel, "*")
```

Modifying this to instead delete all wovels could be another good exercise.

Reply:
ok..